tox.ini

```diff
@@ -12,13 +14,16 @@ deps =
     requests
     # cdfifflib means that benchmarks run faster
     cdifflib
+    dbt017: dbt==0.17.2
+    dbt018: dbt==0.18.1
 # Include any other steps necessary for testing below.
 # {posargs} is there to allow us to specify specific tests, which
 # can then be invoked from tox by calling e.g.
 # tox -e py35 -- project/tests/test_file.py::TestClassName::test_method
 commands =
+    make -C test/ fixtures/dbt_project/dbt_modules
     python util.py clean-tests
-    pytest -vv --cov {posargs}
+    pytest -vv --cov {posargs:-m "not dbt"}

```

Comment:
When I run this locally, both the `py39` and `dbt018-py39` environments deselect the dbt tests - can you double check that these are set up correctly?

Reply:
you need to run like this: 

```bash
tox -e dbt017-py38 -- -m "dbt"
```

I've added instruction in the CONTRIBUTING